### ISSUE TITLE
[ethers-v4] New factories folder causing module resolution problem.

### ISSUE THREAD

**Issue by alexroan** [ethereum-ts/TypeChain#304]
Since [this fix](https://github.com/ethereum-ts/TypeChain/issues/225), I'm getting the following issue inside a generated factory:

```
src/generated/factories/LinkToken__factory.ts(9,38): error TS2307: Cannot find module '.'
```

It appears this line is trying to import TransactionOverrides from the factories folder:

```
import { TransactionOverrides } from ".";
```

This works if the "." becomes ".."

**Comment by quezak** [comment]
Looks like you're right -- we must've missed it since most of us use ethers v5 by now :)
Seems like an easy fix, could you prepare a PR?

**Comment by krzkaczor** [comment]
Released as `@typechain/ethers-v4@3.0.1`
